pkg/handlers/generic/mutation/auditpolicy/inject.go

```diff
@@ -25,7 +26,7 @@ type auditPolicyPatchHandler struct{}
 //go:embed embedded/apiserver-audit-policy.yaml
 var auditPolicy string
 
-const auditPolicyPath = "/etc/kubernetes/audit-policy/apiserver-audit-policy.yaml"
+const auditPolicyPath = "/etc/kubernetes/audit-policy.yaml"
```

Comment:
This will cause a rollout on all existing clusters when CAREN version is upgraded.
We will need to bump the handler version and copy the old implementation to https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/tree/main/pkg/handlers/v2 (but changed to v3 and removed the kube-vip handler from there)

(can be done as a separate PR)

Reply:
Good catch! I'll open a follow up PR before we merge this one.